### ISSUE TITLE
Popover doesn't open after been closed with 'hide' method.

### ISSUE THREAD

**Issue by mariogl** [twbs/bootstrap#19372]
After a popover is closed by method 'hide' from code, it's necessary to click on its trigger twice in order to open it again.

I'm using Bootstrap v3.3.6. If I use previous versions of Bootstrap (i.e., v3.0.2), it works fine. 

An example that illustrates this issue: http://codepen.io/mariogl/pen/EKapXE

**Comment by twbs-lmvtfy** [comment]
Hi @mariogl!

You appear to have posted a live example (http://codepen.io/mariogl/pen/EKapXE.html), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 8, column 1: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 10, column 1: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by mariogl** [comment]
Revised example: http://codepen.io/mariogl/pen/EKapXE

**Comment by cvrebert** [comment]
Duplicate of #16732.
